editor/editor_help.cpp

```diff
@@ -3806,76 +3806,108 @@ EditorHelpBit::EditorHelpBit(const String &p_symbol) {
 	content = memnew(RichTextLabel);
 	content->set_theme_type_variation("EditorHelpBitContent");
 	content->set_custom_minimum_size(Size2(512 * EDSCALE, content_min_height));
-	content->set_selection_enabled(true);
-	//content->set_context_menu_enabled(true); // TODO: Fix opening context menu hides tooltip.
+	content->set_selection_enabled(p_allow_selection);
+	content->set_context_menu_enabled(p_allow_selection);
 	content->connect("meta_clicked", callable_mp(this, &EditorHelpBit::_meta_clicked));
 	add_child(content);
 
 	if (!p_symbol.is_empty()) {
-		parse_symbol(p_symbol);
+		parse_symbol(p_symbol, p_prologue);
 	}
```

Comment:
Despite my best efforts, I still missed one thing. Please add this if you will cherry-pick this for 4.3. For master, this will be fixed in #91060.

```diff
 	if (!p_symbol.is_empty()) {
 		parse_symbol(p_symbol, p_prologue);
+	} else if (!p_prologue.is_empty()) {
+		set_custom_text(String(), String(), p_prologue);
 	}
```